be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/ElectionHandler.scala

```diff
@@ -48,6 +51,7 @@ class ElectionHandler(dbRef: => AskableActorRef) extends MessageHandler {
     val combined = for {
       _ <- dbActor ? DbActor.WriteAndPropagate(rpcMessage.getParamsChannel, message)
       _ <- dbActor ? DbActor.CreateChannel(electionChannel, ObjectType.ELECTION)
+      _ <- dbActor ? DbActor.WriteAndPropagate(electionChannel, message)
     } yield ()
```

Comment:
This was addressed on slack, since both backends intially did not have the setup message written in the channel election, and it was a problem when we wanted to retrieve the setup message. Pierluca affirmed that the setup message should also be included in both the lao and election channel.